src/fmu/dataio/dataio.py

```diff
@@ -203,6 +276,79 @@ def __init__(
 
         logger.info("Create instance of ExportData")
 
+    # ==================================================================================
+    # Private attributes that are or may be exposed read-only for other classes
+
+    @property
+    def runpath(self):
+        """Return runpath which is the relative path to run root folder."""
+        return self._runpath
+
+    @property
+    def context(self):
+        """Return context."""
+        return self._context
+
+    @property
+    def content(self):
+        """Return content."""
+        return self._content
+
+    @property
+    def config(self):
+        """Return config."""
+        return self._config
+
+    @property
+    def subfolder(self):
+        """Return subfolder name."""
+        return self._subfolder
+
+    @property
+    def use_index(self):
```

Comment:
Not sure about the `use_index`. It feels ambiguous. Would `include_index` be better (more descriptive)?